content/docs/tasks/security/rbac-groups/index.md

```diff
@@ -0,0 +1,281 @@
+
+    {{< text bash>}}
+    $ TOKEN=$(curl {{< github_file >}}/security/tools/jwt/samples/groups-scope.jwt -s)
+    {{< /text >}}
+
+1.  Connect to the `httpbin` service:
+
+    {{< text bash >}}
+    $ kubectl exec $(kubectl get pod -l app=sleep -n $NS -o jsonpath={.items..metadata.name}) -c sleep -n $NS -- curl http://httpbin.$NS:8000/ip -s -o /dev/null -w "%{http_code}\n" --header "Authorization: Bearer $TOKEN"
+    {{< /text >}}
+
+    When a valid JWT is attached, it returns the HTTP code 200.
+
+1.  Verify that the connection to the `httpbin` service fails when the JWT is not attached:
+
+    {{< text bash >}}
+    $ kubectl exec $(kubectl get pod -l app=sleep -n $NS -o jsonpath={.items..metadata.name}) -c sleep -n $NS -- curl http://httpbin.$NS:8000/ip -s -o /dev/null -w "%{http_code}\n"
+    {{< /text >}}
+
+    When no valid JWT is attached, it returns the HTTP code 401.
+
+## Configure groups-based authorization
+
+This section creates a policy to authorize the access to the `httpbin`
+service if the requests are originated from specific groups.
+As there may be some delays due to caching and other propagation overhead,
+wait until the newly defined RBAC policy to take effect.
+
+1.  Enable the Istio RBAC for the namespace:
+
+    {{< text bash >}}
+    $ cat <<EOF | kubectl apply -n $NS -f -
+    apiVersion: "rbac.istio.io/v1alpha1"
+    kind: RbacConfig
+    metadata:
+      name: default
+    spec:
+      mode: 'ON_WITH_INCLUSION'
+      inclusion:
+        namespaces: ["rbac-groups-test-ns"]
+    EOF
+    {{< /text >}}
+
+1.  Once the RBAC policy takes effect, verify that Istio rejected the curl
+connection to the `httpbin` service:
+
+    {{< text bash >}}
+    $ kubectl exec $(kubectl get pod -l app=sleep -n $NS -o jsonpath={.items..metadata.name}) -c sleep -n $NS -- curl http://httpbin.$NS:8000/ip -s -o /dev/null -w "%{http_code}\n" --header "Authorization: Bearer $TOKEN"
+    {{< /text >}}
+
+    Once the RBAC policy takes effect, the command returns the HTTP code 403.
+
+1.  To give read access to the `httpbin` service, create the `httpbin-viewer`
+service role:
+
+    {{< text bash >}}
+    $ cat <<EOF | kubectl apply -n $NS -f -
+    apiVersion: "rbac.istio.io/v1alpha1"
+    kind: ServiceRole
+    metadata:
+      name: httpbin-viewer
+      namespace: rbac-groups-test-ns
+    spec:
+      rules:
+      - services: ["httpbin.rbac-groups-test-ns.svc.cluster.local"]
+        methods: ["GET"]
+    EOF
+    {{< /text >}}
+
+1.  To assign the `httpbin-viewer` role to users in `group1`, create the
+`bind-httpbin-viewer` service role binding.
+
+    {{< text bash >}}
+    $ cat <<EOF | kubectl apply -n $NS -f -
+    apiVersion: "rbac.istio.io/v1alpha1"
+    kind: ServiceRoleBinding
+    metadata:
+      name: bind-httpbin-viewer
+      namespace: rbac-groups-test-ns
+    spec:
+      subjects:
+      - properties:
+          request.auth.claims[groups]: "group1"
```

Comment:
You can use 
```
- group: "group1"
```

Reply:
Done: added the alternative method to specify group.